src/assets/css/_sass/components/_site-nav.scss

```diff
@@ -20,4 +22,42 @@
       background-color: -color($greys, 20);
     }
   }
+
+  .site-nav__item.nav__item:last-child {
```

Comment:
the retailers may not always be the last item (It will, but shhh), might be worth exploring how to get item name onto the `.site-nav__item` to make targeting nicer (Not sure if it's doable, so this is fine for now, but just for future).

There shouldn't be a need to target both `.site-nav__item` and `.nav__item` - these are specific styles for the `site-nav` so just the one will do.